devfile.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: 1.0.0
+metadata:
+  generateName: che-theia-
+
+components:
+  - mountSources: true
+    command:
+      - tail
+      - '-f'
+      - /dev/null
+    memoryLimit: 512Gi
```

Comment:
it looks like a big amount of memory ;-)

Reply:
ouch ...

Reply:
![Selection_209](https://user-images.githubusercontent.com/650571/107398697-2df2f000-6b00-11eb-93c8-cc6c1f7da487.png)

Reply:
Fixed, just gave 1Gi, should be fair enough :)